Traceback (most recent call last):
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# file identifier where the model output is saved
file_id = '_ct'
tasks.run_with_hydro(gdir, run_task=tasks.run_constant_climate, nyears=100, y0=2014, halfsize=5, store_monthly_hydro=True, 
                     output_filesuffix=file_id);
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mFileNotFoundError[0m                         Traceback (most recent call last)
Cell [0;32mIn[7], line 3[0m
[1;32m      1[0m [38;5;66;03m# file identifier where the model output is saved[39;00m
[1;32m      2[0m file_id [38;5;241m=[39m [38;5;124m'[39m[38;5;124m_ct[39m[38;5;124m'[39m
[0;32m----> 3[0m [43mtasks[49m[38;5;241;43m.[39;49m[43mrun_with_hydro[49m[43m([49m[43mgdir[49m[43m,[49m[43m [49m[43mrun_task[49m[38;5;241;43m=[39;49m[43mtasks[49m[38;5;241;43m.[39;49m[43mrun_constant_climate[49m[43m,[49m[43m [49m[43mnyears[49m[38;5;241;43m=[39;49m[38;5;241;43m100[39;49m[43m,[49m[43m [49m[43my0[49m[38;5;241;43m=[39;49m[38;5;241;43m2014[39;49m[43m,[49m[43m [49m[43mhalfsize[49m[38;5;241;43m=[39;49m[38;5;241;43m5[39;49m[43m,[49m[43m [49m[43mstore_monthly_hydro[49m[38;5;241;43m=[39;49m[38;5;28;43;01mTrue[39;49;00m[43m,[49m[43m [49m
[1;32m      4[0m [43m                     [49m[43moutput_filesuffix[49m[38;5;241;43m=[39;49m[43mfile_id[49m[43m)[49m;

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/utils/_workflow.py:491[0m, in [0;36mentity_task.__call__.<locals>._entity_task[0;34m(gdir, reset, print_log, return_value, continue_on_error, add_to_log_file, **kwargs)[0m
[1;32m    489[0m     signal[38;5;241m.[39malarm(cfg[38;5;241m.[39mPARAMS[[38;5;124m'[39m[38;5;124mtask_timeout[39m[38;5;124m'[39m])
[1;32m    490[0m ex_t [38;5;241m=[39m time[38;5;241m.[39mtime()
[0;32m--> 491[0m out [38;5;241m=[39m [43mtask_func[49m[43m([49m[43mgdir[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m
[1;32m    492[0m ex_t [38;5;241m=[39m time[38;5;241m.[39mtime() [38;5;241m-[39m ex_t
[1;32m    493[0m [38;5;28;01mif[39;00m cfg[38;5;241m.[39mPARAMS[[38;5;124m'[39m[38;5;124mtask_timeout[39m[38;5;124m'[39m] [38;5;241m>[39m [38;5;241m0[39m:

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/core/flowline.py:3663[0m, in [0;36mrun_with_hydro[0;34m(gdir, run_task, store_monthly_hydro, fixed_geometry_spinup_yr, ref_area_from_y0, ref_area_yr, ref_geometry_filesuffix, **kwargs)[0m
[1;32m   3660[0m [38;5;28;01mif[39;00m fixed_geometry_spinup_yr [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[1;32m   3661[0m     kwargs[[38;5;124m'[39m[38;5;124mfixed_geometry_spinup_yr[39m[38;5;124m'[39m] [38;5;241m=[39m fixed_geometry_spinup_yr
[0;32m-> 3663[0m out [38;5;241m=[39m [43mrun_task[49m[43m([49m[43mgdir[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m
[1;32m   3665[0m [38;5;28;01mif[39;00m out [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[1;32m   3666[0m     [38;5;28;01mraise[39;00m InvalidWorkflowError([38;5;124m'[39m[38;5;124mThe run task ([39m[38;5;132;01m{}[39;00m[38;5;124m) did not run [39m[38;5;124m'[39m
[1;32m   3667[0m                                [38;5;124m'[39m[38;5;124msuccessfully.[39m[38;5;124m'[39m[38;5;241m.[39mformat(run_task[38;5;241m.[39m[38;5;18m__name__[39m))

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/utils/_workflow.py:491[0m, in [0;36mentity_task.__call__.<locals>._entity_task[0;34m(gdir, reset, print_log, return_value, continue_on_error, add_to_log_file, **kwargs)[0m
[1;32m    489[0m     signal[38;5;241m.[39malarm(cfg[38;5;241m.[39mPARAMS[[38;5;124m'[39m[38;5;124mtask_timeout[39m[38;5;124m'[39m])
[1;32m    490[0m ex_t [38;5;241m=[39m time[38;5;241m.[39mtime()
[0;32m--> 491[0m out [38;5;241m=[39m [43mtask_func[49m[43m([49m[43mgdir[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m
[1;32m    492[0m ex_t [38;5;241m=[39m time[38;5;241m.[39mtime() [38;5;241m-[39m ex_t
[1;32m    493[0m [38;5;28;01mif[39;00m cfg[38;5;241m.[39mPARAMS[[38;5;124m'[39m[38;5;124mtask_timeout[39m[38;5;124m'[39m] [38;5;241m>[39m [38;5;241m0[39m:

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/core/flowline.py:3422[0m, in [0;36mrun_constant_climate[0;34m(gdir, nyears, y0, halfsize, bias, temperature_bias, precipitation_factor, store_monthly_step, store_model_geometry, store_fl_diagnostics, init_model_filesuffix, init_model_yr, output_filesuffix, climate_filename, mb_model_class, climate_input_filesuffix, init_model_fls, zero_initial_glacier, **kwargs)[0m
[1;32m   3343[0m [38;5;129m@entity_task[39m(log)
[1;32m   3344[0m [38;5;28;01mdef[39;00m [38;5;21mrun_constant_climate[39m(gdir, nyears[38;5;241m=[39m[38;5;241m1000[39m, y0[38;5;241m=[39m[38;5;28;01mNone[39;00m, halfsize[38;5;241m=[39m[38;5;241m15[39m,
[1;32m   3345[0m                          bias[38;5;241m=[39m[38;5;241m0[39m, temperature_bias[38;5;241m=[39m[38;5;28;01mNone[39;00m,
[0;32m   (...)[0m
[1;32m   3357[0m                          zero_initial_glacier[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[1;32m   3358[0m                          [38;5;241m*[39m[38;5;241m*[39mkwargs):
[1;32m   3359[0m [38;5;250m    [39m[38;5;124;03m"""Runs the constant mass balance model for a given number of years.[39;00m
[1;32m   3360[0m 
[1;32m   3361[0m [38;5;124;03m    This will initialize a[39;00m
[0;32m   (...)[0m
[1;32m   3419[0m [38;5;124;03m        kwargs to pass to the FluxBasedModel instance[39;00m
[1;32m   3420[0m [38;5;124;03m    """[39;00m
[0;32m-> 3422[0m     mb_model [38;5;241m=[39m [43mMultipleFlowlineMassBalance[49m[43m([49m[43mgdir[49m[43m,[49m
[1;32m   3423[0m [43m                                           [49m[43mmb_model_class[49m[38;5;241;43m=[39;49m[43mpartial[49m[43m([49m
[1;32m   3424[0m [43m                                               [49m[43mConstantMassBalance[49m[43m,[49m
[1;32m   3425[0m [43m                                               [49m[43mmb_model_class[49m[38;5;241;43m=[39;49m[43mmb_model_class[49m[43m)[49m[43m,[49m
[1;32m   3426[0m [43m                                           [49m[43my0[49m[38;5;241;43m=[39;49m[43my0[49m[43m,[49m[43m [49m[43mhalfsize[49m[38;5;241;43m=[39;49m[43mhalfsize[49m[43m,[49m
[1;32m   3427[0m [43m                                           [49m[43mbias[49m[38;5;241;43m=[39;49m[43mbias[49m[43m,[49m
[1;32m   3428[0m [43m                                           [49m[43mfilename[49m[38;5;241;43m=[39;49m[43mclimate_filename[49m[43m,[49m
[1;32m   3429[0m [43m                                           [49m[43minput_filesuffix[49m[38;5;241;43m=[39;49m[43mclimate_input_filesuffix[49m[43m)[49m
[1;32m   3431[0m     [38;5;28;01mif[39;00m temperature_bias [38;5;129;01mis[39;00m [38;5;129;01mnot[39;00m [38;5;28;01mNone[39;00m:
[1;32m   3432[0m         mb_model[38;5;241m.[39mtemp_bias [38;5;241m+[39m[38;5;241m=[39m temperature_bias

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/core/massbalance.py:1194[0m, in [0;36mMultipleFlowlineMassBalance.__init__[0;34m(self, gdir, fls, mb_model_class, use_inversion_flowlines, input_filesuffix, **kwargs)[0m
[1;32m   1190[0m     [38;5;28;01melse[39;00m:
[1;32m   1191[0m         rgi_filesuffix [38;5;241m=[39m input_filesuffix
[1;32m   1193[0m     [38;5;28mself[39m[38;5;241m.[39mflowline_mb_models[38;5;241m.[39mappend(
[0;32m-> 1194[0m         [43mmb_model_class[49m[43m([49m[43mgdir[49m[43m,[49m[43m [49m[43minput_filesuffix[49m[38;5;241;43m=[39;49m[43mrgi_filesuffix[49m[43m,[49m
[1;32m   1195[0m [43m                       [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m)
[1;32m   1197[0m [38;5;28mself[39m[38;5;241m.[39mvalid_bounds [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mflowline_mb_models[[38;5;241m-[39m[38;5;241m1[39m][38;5;241m.[39mvalid_bounds
[1;32m   1198[0m [38;5;28mself[39m[38;5;241m.[39mhemisphere [38;5;241m=[39m gdir[38;5;241m.[39mhemisphere

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/core/massbalance.py:702[0m, in [0;36mConstantMassBalance.__init__[0;34m(self, gdir, mb_model_class, y0, halfsize, **kwargs)[0m
[1;32m    685[0m [38;5;250m[39m[38;5;124;03m"""Initialize[39;00m
[1;32m    686[0m 
[1;32m    687[0m [38;5;124;03mParameters[39;00m
[0;32m   (...)[0m
[1;32m    698[0m [38;5;124;03m    keyword arguments to pass to the mb_model_class[39;00m
[1;32m    699[0m [38;5;124;03m"""[39;00m
[1;32m    701[0m [38;5;28msuper[39m()[38;5;241m.[39m[38;5;21m__init__[39m()
[0;32m--> 702[0m [38;5;28mself[39m[38;5;241m.[39mmbmod [38;5;241m=[39m [43mmb_model_class[49m[43m([49m[43mgdir[49m[43m,[49m
[1;32m    703[0m [43m                            [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m
[1;32m    705[0m [38;5;28;01mif[39;00m y0 [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[1;32m    706[0m     [38;5;28;01mraise[39;00m InvalidParamsError([38;5;124m'[39m[38;5;124mPlease set `y0` explicitly[39m[38;5;124m'[39m)

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/core/massbalance.py:384[0m, in [0;36mMonthlyTIModel.__init__[0;34m(self, gdir, filename, input_filesuffix, fl_id, melt_f, temp_bias, prcp_fac, bias, ys, ye, repeat, check_calib_params)[0m
[1;32m    381[0m [38;5;28mself[39m[38;5;241m.[39mgdir [38;5;241m=[39m gdir
[1;32m    383[0m [38;5;28;01mif[39;00m melt_f [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[0;32m--> 384[0m     melt_f [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mcalib_params[49m[[38;5;124m'[39m[38;5;124mmelt_f[39m[38;5;124m'[39m]
[1;32m    386[0m [38;5;28;01mif[39;00m temp_bias [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[1;32m    387[0m     temp_bias [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mcalib_params[[38;5;124m'[39m[38;5;124mtemp_bias[39m[38;5;124m'[39m]

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/utils/_workflow.py:298[0m, in [0;36mlazy_property.<locals>._lazy_property[0;34m(self)[0m
[1;32m    294[0m [38;5;129m@property[39m
[1;32m    295[0m [38;5;129m@wraps[39m(fn)
[1;32m    296[0m [38;5;28;01mdef[39;00m [38;5;21m_lazy_property[39m([38;5;28mself[39m):
[1;32m    297[0m     [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mhasattr[39m([38;5;28mself[39m, attr_name):
[0;32m--> 298[0m         [38;5;28msetattr[39m([38;5;28mself[39m, attr_name, [43mfn[49m[43m([49m[38;5;28;43mself[39;49m[43m)[49m)
[1;32m    299[0m     [38;5;28;01mreturn[39;00m [38;5;28mgetattr[39m([38;5;28mself[39m, attr_name)

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/core/massbalance.py:546[0m, in [0;36mMonthlyTIModel.calib_params[0;34m(self)[0m
[1;32m    543[0m [38;5;129m@lazy_property[39m
[1;32m    544[0m [38;5;28;01mdef[39;00m [38;5;21mcalib_params[39m([38;5;28mself[39m):
[1;32m    545[0m     [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mfl_id [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[0;32m--> 546[0m         [38;5;28;01mreturn[39;00m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mgdir[49m[38;5;241;43m.[39;49m[43mread_json[49m[43m([49m[38;5;124;43m'[39;49m[38;5;124;43mmb_calib[39;49m[38;5;124;43m'[39;49m[43m)[49m
[1;32m    547[0m     [38;5;28;01melse[39;00m:
[1;32m    548[0m         [38;5;28;01mtry[39;00m:

File [0;32m/usr/local/pyenv/versions/3.10.10/lib/python3.10/site-packages/oggm/utils/_workflow.py:3039[0m, in [0;36mGlacierDirectory.read_json[0;34m(self, filename, filesuffix, allow_empty)[0m
[1;32m   3037[0m         out [38;5;241m=[39m {}
[1;32m   3038[0m [38;5;28;01melse[39;00m:
[0;32m-> 3039[0m     [38;5;28;01mwith[39;00m [38;5;28;43mopen[39;49m[43m([49m[43mfp[49m[43m,[49m[43m [49m[38;5;124;43m'[39;49m[38;5;124;43mr[39;49m[38;5;124;43m'[39;49m[43m)[49m [38;5;28;01mas[39;00m f:
[1;32m   3040[0m         out [38;5;241m=[39m json[38;5;241m.[39mload(f)
[1;32m   3041[0m [38;5;28;01mreturn[39;00m out

[0;31mFileNotFoundError[0m: [Errno 2] No such file or directory: '/tmp/OGGM/OGGMHydro/per_glacier/RGI60-11/RGI60-11.00/RGI60-11.00897/mb_calib.json'
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/OGGM/OGGMHydro/per_glacier/RGI60-11/RGI60-11.00/RGI60-11.00897/mb_calib.json'

